pkg/controller/core/clusterqueue_controller_test.go

```diff
@@ -521,6 +522,7 @@ func TestClusterQueuePendingWorkloadsStatus(t *testing.T) {
 		},
 	}
 	for name, tc := range testCases {
+		defer features.SetFeatureGateDuringTest(t, features.ClusterQueueVisibility, true)()
```

Comment:
```suggestion
		defer features.SetFeatureGateDuringTest(t, features.ClusterQueueVisibility, tc.enableQueueVisibility)()
```
Enabling the feature based on `enableQueueVisibility` will allow us to also add a test case where the feature gate is disabled.

Reply:
done https://github.com/kubernetes-sigs/kueue/pull/1139/commits/0284c1110a8b841d28b80befee3fc8869cc6fa09